File: Password.java
import java.util.Scanner;

public class Password{
 public static void main(String[] args){
 Scanner input = new Scanner(System.in);
 
System.out.println("Enter password");
String  userInput = input.nextLine();

if(userInput.length() <= 10 && userInput.length() > 6){
System.out.print("Medium");
}else if(userInput.length() < 6){
System.out.println("weak");
}else if(userInput.length() > 10){
System.out.println("Strong");
}else if(userInput.length() < 1)
System.out.println("This code is invalid");
}
}

File: password.py
#colllect input from users
#save it in a variable
#compare if 


integers = input("password: ")


 
if len(integers) <= 10 and len(integers) > 6:
    print("medium")
elif len(integers) < 6:
    print("Weak")
elif len(integers) > 10:
    print("Strong")
elif len(integers) < 1:
    print("This code is invalid")
